src/scenes/NftDetailPage/NftDetailText.tsx

```diff
@@ -41,6 +34,27 @@ function NftDetailText({ nft }: Props) {
   );
 }
 
+type NftOwnerProps = {
+  owner_address: string;
```

Comment:
nit: would prefer camel case for props, but not a big deal!

Reply:
oh good point I'll fix!